src/Literate.jl

```diff
@@ -260,6 +260,8 @@ function create_configuration(inputfile; user_config, user_kwargs, type=nothing)
                        !get(user_config, "execute", cfg["execute"]) ?
                        ("````@example $(get(user_config, "name", replace(cfg["name"], r"\s" => "_")))" => "````") :
                        ("````julia" => "````")
+    cfg["image_formats"] = [(MIME("image/png"), ".png"), (MIME("image/jpeg"), ".jpeg"),
```

Comment:
I would put SVG first -- Documenter, IJulia, Literate all prioritize "richer" formats when available.

Reply:
Done, also introduced a constant to keep the docs consistent in case this is changed.